source/editableText.py

```diff
@@ -90,7 +94,17 @@ def _hasCaretMoved(self, bookmark, retryInterval=0.01, timeout=None, origWord=No
 				newInfo = self.makeTextInfo(textInfos.POSITION_CARET)
 			except (RuntimeError,NotImplementedError):
 				newInfo = None
-			# Caret events are unreliable in some controls.
+			else:
+				# Caret events are unreliable in some controls.
+				# Only use them if we consider them safe to rely on for a particular control,
+				# and only if they arrive within 20 mili seconds after causing the event to occur.

```

Comment:
This comment says 20 ms but `useEvents_maxTimeoutMs` is 10

Reply:
Ah, I'm sorry. I first hard coded the 20 ms, but then decided to put it into a constant. Just forgot to change it :)